enterprise/backend/enterprise_core/settings.py

```
@@ -11,18 +11,22 @@
 
 from pathlib import Path
 import os
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)